src/components/daydream-controls.js

```diff
@@ -137,21 +146,24 @@ module.exports.Component = registerComponent('daydream-controls', {
     if (!this.everGotGamepadEvent) { this.checkIfControllerPresent(); }
   },
 
-  // No need for onButtonChanged, since Daydream controller has no analog buttons.
-
   onModelLoaded: function (evt) {
     var controllerObject3D = evt.detail.model;
     var buttonMeshes;
     if (!this.data.model) { return; }
     buttonMeshes = this.buttonMeshes = {};
-    buttonMeshes.menu = controllerObject3D.getObjectByName('AppButton_AppButton_Cylinder.004');
-    buttonMeshes.system = controllerObject3D.getObjectByName('HomeButton_HomeButton_Cylinder.005');
-    buttonMeshes.trackpad = controllerObject3D.getObjectByName('TouchPad_TouchPad_Cylinder.003');
-    // Offset pivot point
+    buttonMeshes.menu = controllerObject3D.getObjectByName(
```

Comment:
If needed I think it looks better to break the whole expression to the line below instead of just an argument